### PR TITLE
US18796 : Create new flo PR checker command for drush features-list, contains 'overridden'.

### PR DESCRIPTION
**Pull request by xopoc14**

### CODE REVIEW

**Comment by bleen** [comment]
We really need to create some namespaces for the flo commands.  This would obviously live in the `drupal` namespace. I feel like that is a different issue though.

As for this command, I like it, but it needs tests :)

File: src/flo/Command/FeaturesOverwrittenChecker.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    if ($git_root->getOutput() !== $current_dir->getOutput()) {
+      throw new \Exception("You must run check-features from the git root.");
+    }
+
+    $pull_request = $this->getConfigParameter('pull_request');
+    $path = "{$pull_request['prefix']}-{$pullRequest}.{$pull_request['domain']}";
+    $pr_directories = $this->getConfigParameter('pr_directories');
+    $process = new Process("cd {$pr_directories}{$path}/scripts && bash features-checker.sh");
```
Comment (bleen) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: what is this file (features-checker.sh)? are we introducing a new dependency here?

File: src/flo/Console/Application.php [hunk 2]
```diff
@@ -110,6 +110,7 @@ protected function getDefaultCommands() {
     $commands[] = new Command\Deployments\TagDeploy();
     $commands[] = new Command\Deployments\TagPreRelease();
     $commands[] = new Command\RunScriptCommand();
+    $commands[] = new Command\FeaturesOverwrittenChecker();
```
Comment (bleen) [inline src/flo/Console/Application.php]: this needs to be added to the readme and the help text as well

File: src/flo/Command/FeaturesOverwrittenChecker.php (hunk 1 above)
Comment (bleen) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: ...cant we just use drush?

File: src/flo/Command/FeaturesOverwrittenChecker.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+use flo\Drupal;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Github;
+
+
+class FeaturesOverwrittenChecker extends Command {
```
Comment (ElijahLynn) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: FeaturesOverriddenChecker

File: src/flo/Command/FeaturesOverwrittenChecker.php [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  protected function configure() {
+    $this->setName('check-features')
+      ->setDescription('runs `drush fl` to check features.');
```
Comment (ElijahLynn) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: drush features-list

**Comment by ElijahLynn** [comment]
Referencing https://github.com/NBCUOTS/Publisher7/pull/2224

File: src/flo/Command/FeaturesOverwrittenChecker.php (hunk 4 above)
Comment (bleen) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: ...and I dont see that command actually run anywhere. I assume its run in this mysterious features-checker.sh file...

Comment (ElijahLynn) [inline src/flo/Command/FeaturesOverwrittenChecker.php]: Yeah, that is where it is run. We discussed IRL and we are going to get rid of that file. Not sure how yet but that is the plan.

**Comment by ElijahLynn** [comment]
Closing in favor of #186.

**Pull request closed without merging**